Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Build the binaries in larger image
-FROM docker.io/fortio/fortio.build:v52@sha256:2a85bd3b97d7eaf1718a3fc7a376d52dbe78f01cd8cb3baee87228ad92adf710 as build
+FROM docker.io/fortio/fortio.build:v54@sha256:4775038c3ace753978c8dfd99ebbd23607b61eeb0bf6c2bf2901d0485cc1870c as build
 WORKDIR /build
-COPY . fortio
+COPY --chown=build:build . fortio
```

Comment:
Required to avoid

#0 0.031 fatal: detected dubious ownership in repository at '/build/fortio'                                                                                                                                                    
[...]
#0 8.906 error obtaining VCS status: exit status 128